test/unit/permission-set-test.js

```diff
@@ -259,3 +260,25 @@ test('PermissionSet init from untyped ACL test', function (t) {
     'Permission set should init correctly without acl:Authorization type')
   t.end()
 })
+
+test('PermissionSet save() test', t => {
+  let resourceUrl = 'https://alice.example.com/docs/file1'
+  let aclUrl = 'https://alice.example.com/docs/file1.acl'
+  let isContainer = false
+  let putSpy = sinon.spy()
+  let mockWebClient = {
+    put: putSpy
+  }
+  let ps = new PermissionSet(resourceUrl, aclUrl, isContainer,
+    { rdf, webClient: mockWebClient })
+  ps.save()
+    .then(() => {
+      t.ok(putSpy.calledWith(aclUrl),
```

Comment:
Aren't you also going to test that the correct graph was passed to the web client?